scripts/core/editor3/reducers/spellchecker.jsx

```diff
@@ -47,4 +50,21 @@ const replaceWord = (state, {word, newWord}) => {
  */
 const refreshWord = (state, word) => replaceWord(state, {word: word, newWord: word.text});
 
+/**
+ * @ngdoc method
+ * @name autoSpelchecker
+ * @param {Object} state
+ * @param {Boolean} auto True if the autospellchecker is enabled
+ * @return {Object} returns new state
+ * @description Disable/enable auto mode for spellchecker.
+ */
+const autoSpellchecker = (state, auto) => {
+    const {editorState} = state;
+    const content = editorState.getCurrentContent();
+    const decorators = Editor3.getDecorator(!auto);
```

Comment:
So if the "automatic spellchecker" is enabled, then the internal spellchecker gets disabled here, right? I'm not sure I can figure out what the "automatic spellchecker" is from reading this code